packages/angular-ui-components/src/controllers/toaster/toaster-controller.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subject } from 'rxjs';
+import { takeUntil, tap } from 'rxjs/operators';
+import { ToasterService } from './toaster-controller.service';
+import { IToaster, EToasterType } from '@kelvininc/ui-components';
+
+@Component({ 
+	selector: 'kv-toaster-controller',
+	template: `
+		<div *ngIf="this.header && this.type">
+			<kv-toaster
+				header="header"
+				description="description"
+				ttl="ttl"
+				type="type"
+				(ttlExpired)="this.onTTLExpired"
```

Comment:
remove the `this` here

Reply:
we don't need to enable bind in the props? like `[header]=...`,`[description]=...`

Reply:
`ttlExpired` and `onClickCloseButton` can call the same function, since both do the same job